diff --git a/msvc/config.h b/msvc/config
--- a/msvc/config.h
+++ b/msvc/config.h
@@ -15,34 +15,9 @@
 #pragma warning(disable:6387)
 #endif
 
-/*
- * Embed WinUSB driver files from the following WDK location.
- * If needed, you can obtain the WDK redistributable components from:
- * https://go.microsoft.com/fwlink/p/?LinkID=253170
- * NB: You must also make sure the WDF_VER, COINSTALLER_DIR and X64_DIR
- * match your WinUSB redist directories.
- */
-#ifndef WDK_DIR
-#define WDK_DIR "C:/Program Files (x86)/Windows Kits/8.0"
-#endif
-
-/* WDK WDF coinstaller version */
-#define WDF_VER 1011
-
-/* CoInstaller subdirectory for WinUSB redist files ("winusb" or "wdf") */
-#define COINSTALLER_DIR "wdf"
-
-/* 64bit subdirectory for WinUSB redist files ("x64" or "amd64") */
-#define X64_DIR "x64"
-
 /* embed libusb0 driver files from the following location */
 #ifndef LIBUSB0_DIR
-#define LIBUSB0_DIR "D:/libusb-win32"
-#endif
-
-/* embed libusbK driver files from the following location */
-#ifndef LIBUSBK_DIR
-#define LIBUSBK_DIR "D:/libusbK/bin"
+#define LIBUSB0_DIR "E:/libusb-win32"
 #endif
 
 /* embed user defined driver files from the following location */
